<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addvaluestoroutine</class>
 <widget class="QDialog" name="Addvaluestoroutine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Value</string>
  </property>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Click here if there is no class in this time</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditrno">
      <property name="placeholderText">
       <string>Enter room no.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditsname">
      <property name="placeholderText">
       <string>Enter subject name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdittname">
      <property name="placeholderText">
       <string>Enter teacher's name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
